docs/quantum-consciousness.org

```diff
@@ -0,0 +1,197 @@
+
+While the notion of these principles is well-defined, the properties ascribed to consciousness are self-consistent, consistent with what we already know with the world, and as I demonstrated earlier, experimentally verifiable, not practically, but in principle!  So there is a cosmic conspiracy, a *philosophical censorship* if you will.
+
+** Beyond philosophical censorship
+
+In Physics, similar conspiracies were often an indication that our previous models were inadequate; we couldn't just alter our knowledge, we needed to change what we operated on.
+
+One way is like Dirac, to admit defeat: to state that consciousness is fundamentally not intelligible from the point of view of science, and that it can only be interrogated as something that manifestly is part of our physical reality, that it can effect change in the physical reality, but not /of physical reality/, there are no positive physical laws that would dictate how a consciousness operates.
+
+This approach is fine.  I suppose it is not satisfying but not counter-productive either.  This also gives a natural exception to free will, so we can immediately chalk that question up to whatever branch of philosophy is going to interrogate consciousness.  In a perverse way it is a status quo for the subject, unproductive though it is.
+
+The second approach we might take, is to build a theory around the fact that this information is fundamental.  This appears to be a sound basis, but if not careful devolves into absolute solipsism.  Indeed, consider that we impose as a fundamental restriction that this information must either be universally attainable, or universally unattainable.  One possibility is that we do not exist: universally unattainable, and thus nothing else does.  We are all philosophical zombies.  Obviously, no objective counter-argument can be made, because any argument you would make against that can be traced to "natural roots".  Yet, /you/ know that you exist as a qualia.  The exact opposite of this principle is the basis of "I think, therefore I am".  The qualia of having thoughts and integrated information is a fact.  A subjective one, possibly, but a fact nonetheless.
```

Comment:
So "existing as a qualia" is a proper way to discuss this term? It's not about separate senses and the experience, but "the whole bunch of senses poured into a subjective experience"?

Anyway, another funny part here is that we're never talking about "partial zombies". IIRC, the brain needs a lot of power to do its thing. And maybe some power-saving is involved not only when we sleep. If it's not activating when we're around other people or think of surveillance, but things are mundane and repetitive, no one checks for the lack of qualia.

Despite apparent terminological incorrectness, I'll allow myself a silly joke: "Did a zombie eat a hiker's brain in the forest if there are no witnesses?" 

```suggestion
The second approach we might take, is to build a theory around the fact that this information is fundamental.  This appears to be a sound basis, but if not careful, it devolves into absolute solipsism.  Indeed, consider that we impose as a fundamental restriction that this information must either be universally attainable, or universally unattainable.  One possibility is that we do not exist: universally unattainable, and thus nothing else does.  We are all philosophical zombies.  Obviously, no objective counter-argument can be made, because any argument you would make against that can be traced to "natural roots".  Yet, /you/ know that you exist as a qualia.  The exact opposite of this principle is the basis of "I think, therefore I am".  The qualia of having thoughts and integrated information is a fact.  A subjective one, possibly, but a fact nonetheless.
```